wasp/apps/fliproll.py

```diff
@@ -0,0 +1,118 @@
+ 
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2020 Daniel Thompson
+# Copyright (C) 2021 Evan Rodgers 'setLillie'
+
+"""Digital Coin Flip
+~~~~~~~~~~~~~~~~
```

Comment:
I think this is one of the reasons why the checks are not successful. I had the same problem, and it wouldn't run successfully until the docstring (`~`) was the same length as the title. You can see the error in the build [here](https://github.com/daniel-thompson/wasp-os/pull/178/checks?check_run_id=2059068256#step:8:51).